aws/table_aws_route53_vpc_association_authorization.go

```diff
@@ -0,0 +1,144 @@
+package aws
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/route53"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+func tableAwsRoute53VPCAssociationAuthorization(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_route53_vpc_association_authorization",
+		Description: "AWS Route53 VPC Association Authorization",
+		List: &plugin.ListConfig{
+			Hydrate: listVPCAssociationAuthorization,
+			Tags:    map[string]string{"service": "route53", "action": "ListVPCAssociationAuthorizations"},
+			KeyColumns: plugin.KeyColumnSlice{
+				{
+					Name:    "hosted_zone_id",
```

Comment:
Hi @jramosf, Instead of making the `hosted_zone_id` as required qual, can we make it as optional quals and use the table `aws_route53_domain` as parent of this table?
Reference: https://github.com/turbot/steampipe-plugin-aws/blob/f4578bfe1aef0c4717a0cbfc71cbbe5421e05aee/aws/table_aws_route53_record.go#L30